Autopilot/PathManager/src/CoordinatedTurns.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * Author: Anthony Berbari
+ */
+
+#include "CoordinatedTurns.hpp"
+#include "PID.hpp"
+
+#include <math.h>
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+#define RUDDER_SCALING_FACTOR 0.8f // should be experimentally determined
+
+#define DEG_TO_RAD(angleInDegrees) ((angleInDegrees) * M_PI / 180.0)
```

Comment:
I'm defining this exact same macros (function? what is it called) in my module. We should put this in a more global .hpp file when the time comes for integration.

Reply:
yea, a macro. Good point, we've got common modules, should keep in mind